### PR TITLE
feat(daemon): remove workspace id from logging requests

### PR DESCRIPTION
**Pull request by jacobowitz**
This remove the workspace_id as a parameter from the log requests. The workspace id is found automatically by checking the entity id and then looking up the workspace in the correct store.

### CODE REVIEW

File: daemon/api/endpoints/logs.py [hunk 1]
```diff
@@ -31,6 +39,16 @@ async def _export_logs(workspace_id: DaemonID, log_id: DaemonID):
         return FileResponse(filepath)
 
 
+def _get_log_file_path(log_id):
```
Comment (deepankarm) [inline daemon/api/endpoints/logs.py:42]: Please move this to helper

File: daemon/stores/__init__.py [hunk 2]
```diff
@@ -1,11 +1,28 @@
...
+from ..models import DaemonID
+from ..models.enums import IDLiterals
 
 pea_store: PeaStore = PeaStore.load()
 pod_store: PodStore = PodStore.load()
 flow_store: FlowStore = FlowStore.load()
 workspace_store: WorkspaceStore = WorkspaceStore.load()
 
+
+def get_store_from_id(entity_id: DaemonID) -> BaseStore:
```
Comment (deepankarm) [inline daemon/stores/__init__.py:13]: Please rebase with `daemon-2.0` branch here

File: tests/daemon/unit/api/endpoints/test_logs.py [hunk 3]
```diff
@@ -13,50 +16,66 @@
...
+        workspace_id=workspace_id,
+    )
 
 
-def test_logs_invalid_flow(fastapi_client):
-    response = fastapi_client.get(f'/logs/{workspace_id}/{nonexisting_id}')
-    assert response.status_code == 404
+def setup_module():
+    print('setup', get_workspace_path(workspace_id, flow_id))
+    _create_flow()
```
Comment (deepankarm) [inline tests/daemon/unit/api/endpoints/test_logs.py:57]: Why is this needed?

Comment (jacobowitz) [reply tests/daemon/unit/api/endpoints/test_logs.py:57]: the new log endpoint is looking up the workspace id from the entity (pod/pea/flow) id. So if it does not exist it will fail.

so it does basically
`correct_store[log_id].workspace_id`

**Comment by jacobowitz** [comment]
rebased against daemon-2.0
